Cargo.toml

```diff
@@ -41,3 +41,6 @@ features = ["use-dev-tty"]
 version = "4"
 features = ["derive"]
 
+[features]
+default = ["sqlite-bundled"]
```

Comment:
This says that the `sqlite-bundled` feature is enabled by default, so it will behave the same way as before. I set it mainly to maintain backward compatibility, i.e. to not break existing build scripts that assume bundled sqlite. However, the default features can be disabled via `--no-default-features` build option (`cargo build --no-default-features`).